src/lib/infusion/src/framework/preferences/messages/speak.json

```diff
@@ -0,0 +1,4 @@
+{
+    "speakLabel": "Text-to-Speech",
+    "speakChoiceLabel": "Enable the page to speak itself out loud."
+}
```

Comment:
This file is actually in Infusion, the change should probably be made there instead.